test/distributed/elastic/launchers/bin/test_script.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```

Comment:
nit: existing tests in the distributed folder use the following convention: 1) file with a `test_` prefix are entry points of tests and is registered in `test/run_test.py`. And we can use things like `pytest test/distributed/rpc/test_tensorpipe_agent.py -k test_func_name -vs` to run individual tests. 2) if some test implementation needs to shared across different test runs, we create a file with `_test` for implementation and import that in multiple `test_` prefix files, e.g., `rpc_test.py` is imported by both `test_tensorpipe_agent.py` and `test_process_group_agent.py`.

I am not sure if makes sense to follow the same convention here, but might be good to do so if it's not too much efforts.